src/sentry/migrations/0099_fix_project_platforms.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-09-02 21:33
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations
+from django.db.models import Q
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def fix_project_platform(apps, schema_editor):
+    """
+    Find projects whose platform is either 'python-tracing' or 'node-tracing',
+    and change them to be either 'python' or 'node' respectively.
+    """
+    Organization = apps.get_model("sentry", "Organization")
+    Project = apps.get_model("sentry", "Project")
+
+    for org in RangeQuerySetWrapperWithProgressBar(Organization.objects.filter(status=0)):
```

Comment:
This was a requirement for my specific backfill a while ago. If you don't care about applying this to all projects in an org at once you can just simplify and iterate over projects.

Reply:
I took one of your migrations as a starting example. 

Would the following be preferable?

```
for project in RangeQuerySetWrapperWithProgressBar(Project.objects.all()):
    # ...
```

Reply:
Yeah, this is the typical pattern, just keeps things simple.